src/clj/rems/workflow/dynamic.clj

```diff
@@ -19,33 +19,33 @@
    :handlers [UserId]})
 
 (def States
-  #{::approved
-    ::closed
-    ::draft
-    ::rejected
-    ::returned
-    ::submitted
-    #_::withdrawn}) ; TODO withdraw support?
+  #{:application.state/approved
+    :application.state/closed
+    :application.state/draft
+    :application.state/rejected
+    :application.state/returned
+    :application.state/submitted
+    #_:application.state/withdrawn}) ; TODO withdraw support?
 
 (def CommandTypes
-  #{#_::accept-license
-    #_::require-license
-    ::accept-invitation
-    ::add-member
-    ::invite-member
-    ::approve
-    ::close
-    ::comment
-    ::decide
-    ::reject
-    ::request-comment
-    ::request-decision
-    ::remove-member
-    ::return
-    ::save-draft
-    ::submit
-    ::uninvite-member
-    #_::withdraw})
+  #{#_:application.command/accept-license
+    #_:application.command/require-license
+    :application.command/accept-invitation
+    :application.command/add-member
+    :application.command/invite-member
+    :application.command/approve
+    :application.command/close
+    :application.command/comment
+    :application.command/decide
+    :application.command/reject
+    :application.command/request-comment
+    :application.command/request-decision
+    :application.command/remove-member
+    :application.command/return
+    :application.command/save-draft
+    :application.command/submit
+    :application.command/uninvite-member
+    #_:application.command/withdraw})
```

Comment:
hmm, states and command types _could_ be workflow specific. are we sure they're not?

Reply:
ok I guess it's a valid design